### PR TITLE
Update installation.md

### PR DESCRIPTION
**Pull request by ssantos21**
Updated installation.md to include the most recent version and installation of the `grpcio-tools` (I included only in the Ubuntu section, which I tested).

### CODE REVIEW

File: doc/getting-started/getting-started/installation.md [hunk 1]
```diff
@@ -134,6 +134,7 @@ To build cln to just install a tagged or master version you can use the followin
 pip3 install --upgrade pip
 pip3 install mako
 pip3 install -r plugins/clnrest/requirements.txt
+pip3 install grpcio-tools
```
Comment (vincenzopalazzo) [inline doc/getting-started/getting-started/installation.md:137]: mh this is strange, it is required to install core lightning? `grpcio-tools` should be used only in tests

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]

File: doc/getting-started/getting-started/installation.md (hunk 1 above)
Comment (ssantos21) [reply doc/getting-started/getting-started/installation.md:137]: Without this, the build fails.

Comment (vincenzopalazzo) [reply doc/getting-started/getting-started/installation.md:137]: Mh this is nice to know :/

Maybe clnrest -> pyln-client -> uses the girpcio packages and maybe the `clnrest/requirements.txt` is not completed.

idk but this required more exploration

Comment (cdecker) [reply doc/getting-started/getting-started/installation.md:137]: It should only need `grpcio-tools` when the `.proto` files have a newer timestamp than the corresponding `_pb2.py` files. Might be a missing generation update?

Comment (cdecker) [reply doc/getting-started/getting-started/installation.md:137]: This is using the release tag? If that fails requiring grpcio-tools then it is likely correct to keep those instructions in there.

**Comment by cdecker** [comment]
ACK https://github.com/ElementsProject/lightning/pull/6989/commits/f480653a6d294271ec75f4c883a3f5441e77ade4

**Pull request merged**